packages/viewer/package.json

```diff
@@ -54,6 +54,7 @@
   "dependencies": {
     "@speckle/objectloader": "workspace:^",
     "@types/flat": "^5.0.2",
+    "@types/underscore": "1.11.15",
```

Comment:
Are we really using underscore somewhere? 🤔

Reply:
Yeah, it's used once, but it's probably not strictly necessary